src/components/Admin/InfoCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const InfoCard = ({
+  name,
+  value,
+  description,
+  children,
+}: {
+  name: string;
+  value?: string;
+  description?: string;
+  children?: JSX.Element;
```

Comment:
```suggestion
  children?: React.ReactNode;
```

Reply:
**question:** why everything is optional here?

Reply:
Because if it isn't provided the layout can deal with it.